src/app/Home/components/HomeComponent.js

```diff
@@ -1,7 +1,67 @@
-import React from 'react';
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
-const HomeComponent = () => (
-  <h1 className='home__title'>Welcome to Andela Societies </h1>
-);
+import { connect } from 'react-redux';
 
-export default HomeComponent;
+export class App extends Component {
```

Comment:
Don't forget to put this back to homecomponent

Reply:
I have reverted to HomeComponent